pkg/printers/internalversion/describe.go

```diff
@@ -1128,7 +1128,8 @@ func printCSIPersistentVolumeSource(csi *api.CSIPersistentVolumeSource, w Prefix
 		"    Driver:\t%v\n"+
 		"    VolumeHandle:\t%v\n"+
 		"    ReadOnly:\t%v\n",
-		csi.Driver, csi.VolumeHandle, csi.ReadOnly)
+		"	 VolumeAttributes:\t%v\n",
```

Comment:
I don't know if this format is going to look good, because the Volume Attributes is a map.  You may need to do something similar to how labels and annotations are printed.